preview-docs/Webhooks.md

```diff
@@ -0,0 +1,79 @@
+# RFC: Webhooks for schema change notifications
+
+To help you better integrate Apollo Studio into your unique workflow, we are starting some work to offer standard webhooks for certain events. The first of those events will be schema changes - any time we detect a change to the schema for your graph.
+
+We are looking for your feedback!
+
+If you have a related need that is not captured below, or if you see things we might be missing or problematic, please engage with us and ask questions through issues on this repository.
+
+## Plan restrictions
+
+Webhooks are available only to customers on our Enterprise plan.
+
+## How this works
+
+From our Integrations page you can create new Webhooks channels and select which types of events you want sent to your integration endpoint. Today we're launching with support for **Schema Change Events**. 
+
+### Terms
+**Event:** Something that happens in Apollo Studio.
+
+**Event Type:** Events have specific types that denote what happened.
+
+At launch we will include two distinct event types: `SCHEMA__PUBLISHED` and `SCHEMA__CHECKED`.
```

Comment:
Is this true or will we only have `SCHEMA__PUBLISHED`? (also why the double underscore?)

Reply:
Oh good question. The architecture doc mentioned both so I figured we were including both. I can remove schema checked.